backend/scripts/ingest.py

```
@@ -0,0 +1,52 @@
+
+def chunk_text(text: str) -> list[str]:
+    chunks = []
+    start = 0
+    while start < len(text):
+        end = start + QDRANT_CHUNK_SIZE
+        chunks.append(text[start:end])
+        start = end - QDRANT_OVERLAP
+    return chunks
+
+def get_embedding(text: str) -> list[float]:
+    r = requests.post(
```

Ruff (0.15.5)

[error] Probable use of `requests` call without timeout

(S113)